projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -81,6 +81,10 @@ export function NewsletterNotice( {
 	showMisconfigurationWarning,
 	isPostPublishPanel = false,
 } ) {
+	const hasPostBeenPublished = useSelect( select =>
+		select( 'core/editor' ).isCurrentPostPublished()
```

Comment:
What do you think about importing `editorStore` instead of hardcoding the store name here? It may make long-term maintenance easier on us.

```diff
diff --git a/projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js b/projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js
index 47320dc03b..44fd250015 100644
--- a/projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js
+++ b/projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js
@@ -13,7 +13,7 @@ import {
 } from '@wordpress/components';
 import { useInstanceId } from '@wordpress/compose';
 import { useSelect } from '@wordpress/data';
-import { PostVisibilityCheck } from '@wordpress/editor';
+import { PostVisibilityCheck, store as editorStore } from '@wordpress/editor';
 import { createInterpolateElement } from '@wordpress/element';
 import { __, sprintf } from '@wordpress/i18n';
 import { META_NAME_FOR_POST_LEVEL_ACCESS_SETTINGS, accessOptions } from './constants';
@@ -82,7 +82,7 @@ export function NewsletterNotice( {
 	isPostPublishPanel = false,
 } ) {
 	const hasPostBeenPublished = useSelect( select =>
-		select( 'core/editor' ).isCurrentPostPublished()
+		select( editorStore ).isCurrentPostPublished()
 	);
 
 	// Get the reach count for the access level

```

Reply:
Thanks for the feedback! Fixed in [5d89184](https://github.com/Automattic/jetpack/pull/30883/commits/5d89184acf155b4e5bc512ee9e0de1166533afe2)